resources/light/samVisualize/primaryResources.json

```diff
@@ -0,0 +1,16 @@
+[
+    "AWS::CodeBuild::Project",
+    "AWS::DynamoDB::Table",
+    "AWS::EC2::Instance",
+    "AWS::Kinesis::Stream",
+    "AWS::Lambda::Function",
+    "AWS::S3::Bucket",
+    "AWS::SNS::Topic",
+    "AWS::SQS::Queue",
+    "AWS::Serverless::Api",
+    "AWS::Serverless::Function",
+    "AWS::Serverless::HttpApi",
+    "AWS::Serverless::SimpleTable",
+    "AWS::Serverless::StateMachine",
+    "AWS::StepFunctions::StateMachine"
+]
```

Comment:
very nitpicking: usually I wouldn't put an array itself as a JSON because it cannot be extended. Imagine in the future we have more types of resources, we would need to add new JSON files.

Suggest:
- rename this file to `resources.json`
- wrap the array in `{"primaryResources": [...]}

some reading: https://stackoverflow.com/questions/19623339/is-a-list-array-valid-json

Reply:
Right, that makes sense. Fixed!